.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+- 
+- 
+- 
+
+Fixes #(issue), Fixes hypertrace/repo#(issue)
+Connects #(issue)
+
+### Type of change
```

Comment:
Not sure we need this checklist. I think the dots described above would provide this information.

Reply:
ohh! forgot to remove. removing this